Dockerfile

```diff
@@ -18,6 +21,7 @@ FROM eclipse-temurin:17-jdk-alpine
 RUN apk update && apk upgrade --no-cache libcrypto3 libssl3
 # fix CVE-2023-52425
 RUN apk upgrade --no-cache libexpat
+RUN apk add --no-cache su-exec
```

Comment:
Is there a strong reason using `su-exec` instead of 'su'?

Reply:
`su-exec` is an idiomatic way to switch users in Docker containers.

Postgres uses it: https://github.com/docker-library/postgres/blob/1424abf76f421d6f7bf933d9e42bbbed866fae3a/12/alpine3.18/docker-entrypoint.sh#L313

Redis uses `gosu`, which is go-implementation of su-exec: https://github.com/docker-library/redis/blob/master/7.2/alpine/docker-entrypoint.sh#L13

`su` is a much more complicated tool. An equivalent would look like this:

```sh
exec su appuser -s /bin/sh -c 'exec "/app/bin/aidial-core" "$@"' _ "$@"
```

which is far from understandable.

Reply:
ok. thx